src/status_im/data_store/realm/schemas/base/v7/core.cljs

```diff
@@ -0,0 +1,12 @@
+(ns status-im.data-store.realm.schemas.base.v7.core
```

Comment:
why you didn't add it to `status-im.data-store.realm.schemas.base.core` ?

Reply:
fixed